src/entities/createPool/createPoolV3/stableSurge/createStableSurge.ts

```diff
@@ -22,6 +22,11 @@ export class CreatePoolStableSurge implements CreatePoolBase {
     private encodeCall(input: CreatePoolStableSurgeInput): Hex {
         const sortedTokenConfigs = sortByAddress(input.tokens);
 
+        const tokens = sortedTokenConfigs.map(({ address, ...rest }) => ({
+            token: address,
+            ...rest,
+        }));
```

Comment:
What triggered these refactors as part of this pr? Simply improving redeability? It seems like these changes are to mimic the way John initially handled arguments.